test/__tests__/data/campaign_manager.json

```diff
@@ -351,10 +345,309 @@
+          },
+          "library": {
+            "name": "RudderLabs JavaScript SDK",
+            "version": "1.0.0"
+          },
+          "userAgent": "Mozilla/5.0 (Macintosh; Intel Mac OS X 10_14_5) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/77.0.3865.90 Safari/537.36",
+          "locale": "en-US",
+          "ip": "0.0.0.0",
+          "os": {
+            "name": "",
+            "version": ""
+          },
+          "screen": {
+            "density": 2
+          }
+        },
+        "originalTimestamp": "2022-11-17T00:22:02.903+05:30",
+        "properties": {
+          "profileId": "34245",
+          "floodlightConfigurationId": "213123123",
+          "ordinal": "1",
+          "floodlightActivityId": "456543345245",
+          "value": "756",
+          "quantity": "455678",
+          "encryptionSource": "AD_SERVING",
+          "encryptionEntityId": "3564523",
+          "encryptionEntityType": "DCM_ACCOUNT",
+          "requestType": "batchinsert",
+          "matchId": "123"
+        },
+        "type": "track",
+        "event": "event test",
+        "anonymousId": "randomId",
+        "integrations": {
+          "All": true
+        },
+        "name": "ApplicationLoaded",
+        "sentAt": "2019-10-14T11:15:53.296Z"
+      },
+      "metadata": {
+        "secret": {
+          "access_token": "dummyApiToken",
+          "refresh_token": "efgh5678",
+          "developer_token": "ijkl91011"
+        }
+      },
+      "destination": {
+        "Config": {
+          "profileId": "5343234",
+          "treatmentForUnderage": false,
+          "limitAdTracking": false,
+          "childDirectedTreatment": false,
+          "nonPersonalizedAd": false,
+          "rudderAccountId": "2EOknn1JNH7WK1MfNku4fGYKkRK"
+        }
+      }
+    },
+    "output": {
+      "version": "1",
+      "type": "REST",
+      "method": "POST",
+      "endpoint": "https://dfareporting.googleapis.com/dfareporting/v4/userprofiles/34245/conversions/batchinsert",
+      "headers": {
+        "Authorization": "Bearer dummyApiToken",
+        "Content-Type": "application/json"
+      },
+      "params": {},
+      "body": {
+        "JSON": {
+          "kind": "dfareporting#conversionsBatchInsertRequest",
+          "conversions": [
+            {
+              "floodlightConfigurationId": "213123123",
+              "ordinal": "1",
+              "timestampMicros": "1668624722903000",
+              "floodlightActivityId": "456543345245",
+              "quantity": "455678",
+              "value": 756,
+              "limitAdTracking": false,
+              "childDirectedTreatment": false,
+              "treatmentForUnderage": false,
+              "nonPersonalizedAd": false,
+              "matchId": "123"
+            }
+          ]
+        },
+        "JSON_ARRAY": {},
+        "XML": {},
+        "FORM": {}
+      },
+      "files": {}
+    }
+  },
+  {
+    "description": "Track - batch insert Call",
```

Comment:
This name matches with other one. Can we try to name based on scenario more descriptively ?